### ISSUE TITLE
Travis Build

### ISSUE THREAD

**Issue by tdipisa** [geosolutions-it/pyfulcrum#11]
A travis build for this project should be configured. This is useful to run unit tests and check PR and commits as usual.

**Comment by cezio** [comment]
for the record: I tried to set up CI job on Travis, but there's a postgis/gdal package incompatibility, for which I couldn't find any solution yet.

The problem is pyfulcrum requires gdal 2.x bindings, because of py3 compatibility. Travis uses old Ubuntu which has gdal 1.11 only, and postgis is linked to that version. I tried to use gdal from ubuntugis PPA (https://launchpad.net/~ubuntugis/+archive/ubuntu/ppa). They have more recent gdal versions (2.1, 2.2 depending on branch), but still postgis is refering gdal 1.x, so there's installation conflict, and gdal 2.x can't be installed with any postigs version from package.

 This requires some more investigation.

Alternatively, to have a build status for PRs, we can use Jenkins: https://github.com/jenkinsci/github-autostatus-plugin
